src/lib/components/FormModel/children/Form/UI/DateTimePicker/date-time-picker.component.js

```diff
@@ -37,29 +36,33 @@ const DateTimePicker = React.memo(
     const label = rest[UI_LABEL] || '';
     const type = rest[RDF_TYPE];
 
-    const updateDate = useCallback(() => {
-      const actualValue = formObject[id] || formObject[id] === '' ? formObject[id].value : value;
+    useEffect(() => {
+      let actualValue = formObject[id] || formObject[id] === '' ? formObject[id].value : value;
```

Comment:
Can we add a comment to this explaining what it's doing in the next few lines?

Reply:
Added a comment with a high level explanation of the logic.